xzre.lds.in

```diff
@@ -69,3 +74,11 @@ SECTIONS_END(.bss)
 SECTIONS_BEGIN()
 	DEFSYM(fake_lzma_allocator, .data.rel.ro.decoders0)
 SECTIONS_END(.data.rel.ro)
+
```

Comment:
@smx-smx I need some help working out how to make these .rodata items - I am not familiar with linking scripts

Reply:
You need to make a new SECTIONS_BEGIN() ... SECTIONS_END(.rodata) block, since it doesn't exist yet. All entries within these lines will be appended to .rodata in the produced file.

Reply:
thanks I think I got it working